src/features/bookOffer/components/Calendar/Calendar.tsx

```diff
@@ -25,12 +26,25 @@ LocaleConfig.locales['fr'] = {
 }
 LocaleConfig.defaultLocale = 'fr'
 
+export const formatToKeyDate = (ISODate: string) => {
```

Comment:
```suggestion
export const formatToKeyDate = (date: Date) => {
```

comme ça tu peux supprimer `const date = new Date(ISODate)`.

Vois si tu peux pas déplacer cette fonction dans un fichier plus adapté.

Reply:
ok je modifie